tests/lean/1079.lean.expected.out

```diff
@@ -1,5 +1,5 @@
 1079.lean:3:2-6:12: error: alternative 'isFalse' has not been provided
-[Meta.Tactic.simp.rewrite] h:1000, m ==> n
+[Meta.Tactic.simp.rewrite] _uniq.82:1000, m ==> n
```

Comment:
What's happening is that the `ToFormat` instance for `SimpLemma` doesn't have the context to look fvars up in the local context. Previously the name of the simp lemma was the user name of the hypothesis, but this can be ambiguous. It seemed relatively unimportant to me so I let it be.

Reply:
This is a trace message; it would be better to do `{mkFVar name}` or `{mkConst name}` anyhow (depending on what it is) because then you can hover over it, get go-to-definition, etc.

Reply:
Is this trace stuff run even when the trace is disabled? The intention here is for the simp lemmas to carry the minimum necessary for reconstruction so as not to slow `simp` down, but if it does all this computation in trace messages it could potentially be a performance issue.

Reply:
The code looks like this:
```lean
trace[Meta.Tactic.simp.rewrite] "{thm}, {e} ==> {rhs}"
```
The `trace[cls] ...` macro expands to `if ← isTracingEnabledForClass cls then ...` so the message is only evaluated if traces are enabled.